bin/reth/src/args/rpc_server_args.rs

```diff
@@ -440,24 +440,43 @@ impl RethRpcConfig for RpcServerArgs {
         Ok(AuthServerConfig::builder(jwt_secret).socket_addr(address).build())
     }
 
-    fn auth_jwt_secret(&self, default_jwt_path: PathBuf) -> Result<JwtSecret, JwtError> {
+    // fn auth_jwt_secret(&self, default_jwt_path: PathBuf) -> Result<JwtSecret, JwtError> {
+    //     match self.auth_jwtsecret.as_ref() {
+    //         Some(fpath) => {
+    //             debug!(target: "reth::cli", user_path=?fpath, "Reading JWT auth secret file");
+    //             JwtSecret::from_file(fpath)
+    //         }
+    //         None => {
+    //             if default_jwt_path.exists() {
+    //                 debug!(target: "reth::cli", ?default_jwt_path, "Reading JWT auth secret
+    // file");                 JwtSecret::from_file(&default_jwt_path)
+    //             } else {
+    //                 info!(target: "reth::cli", ?default_jwt_path, "Creating JWT auth secret
+    // file");                 JwtSecret::try_create(&default_jwt_path)
+    //             }
+    //         }
+    //     }
+    // }
+    ///
```

Comment:
old comment should be removed

Reply:
> old comment should be removed

Done, sorry for this. I forget to take it off